src/lib/components/workflow/workflows-summary-configurable-table/table-empty-state.svelte

```diff
@@ -74,11 +78,13 @@
       </ul>
     {/if}
   </div>
-  <div class="bg-[#DDD6FE]">
+  <div
+    class="flex h-64 items-center justify-center overflow-hidden bg-[#DDD6FE] xl:h-auto xl:flex-1"
+  >
     <img
       src={query ? noResultsImages : emptyImage}
```

Comment:
We're using the same image regardless of query, so we can simplify this 
```suggestion
      src={emptyImage}
```